src/main/java/org/recordrobotics/charger/RobotContainer.java

```diff
@@ -7,10 +7,15 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.recordrobotics.charger.commands.manual.ManualClaw;
 import org.recordrobotics.charger.commands.manual.ManualDrive;
-import org.recordrobotics.charger.control.DoubleControl;
+//import org.recordrobotics.charger.commands.manual.ManualDrive;
```

Comment:
Remove commented statements here

Reply:
why is that even commented? is it not imported above?

Reply:
done